inference-engine/src/inference_engine/src/threading/ie_tbb_streams_executor.cpp

```diff
@@ -65,7 +65,9 @@ struct TBBStreamsExecutor::Impl {
             void on_scheduler_entry(bool) override {
                 _localStream->local() = _thisStream;
                 if (nullptr != _mask) {
-                    PinThreadToVacantCore(_offset + tbb::this_task_arena::current_thread_index(),
+                    PinThreadToVacantCore(_offset + ((tbb::this_task_arena::max_concurrency() > 1)
+                                                         ? tbb::this_task_arena::current_thread_index()
+                                                         : 0),
```

Comment:
Could you reuse the `parallel_get_thread_num` method from `ie_parallel.hpp` here?

Reply:
done